tests/rustfmt/main.rs

```diff
@@ -106,3 +106,13 @@ fn inline_config() {
             && contains("format_strings = true")
     );
 }
+
+#[test]
+fn rustfmt_help() {
```

Comment:
a little nit picky, but this isn't really a full test of the help text, its more specifically a test of the usage text. Would it be possible to rename this to be more specific?

Reply:
sure will rename it